integration-cli/docker_cli_by_digest_test.go

```diff
@@ -341,6 +344,7 @@ func (s *DockerRegistrySuite) TestListDanglingImagesWithDigests(c *testing.T) {
 	// make sure repo shown, tag=<none>, digest = $digest1
 	re1 := regexp.MustCompile(`\s*` + repoName + `\s*<none>\s*` + digest1.String() + `\s`)
 	assert.Assert(c, re1.MatchString(out), "expected %q: %s", re1.String(), out)
+
```

Comment:
oh! unrelated change (if you still have the branch open locally)

Reply:
Fixed, sorry! 😄

Reply:
No worries! Thought you still had the branch at hand, so would be easy to fix (and wouldn't have been a massive disaster if it was left)